boards/phytec/mimx93_phyboard_nash/mimx93_phyboard_nash_a55.dts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022,2024 NXP
+ * Copyright 2024 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mimx93_a55.dtsi>
+#include "mimx93_phyboard_nash-pinctrl.dtsi"
+
+/ {
+	model = "PHYTEC phyBOARD-Nash i.MX93";
+	compatible = "phytec,mimx93_phyboard_nash";
+
+	chosen {
+		zephyr,console = &lpuart2;
+		zephyr,shell-uart = &lpuart2;
+		zephyr,sram = &sram0;
+	};
+
+	cpus {
+		cpu@0 {
+			status = "disabled";
+		};
+	};
+
+	sram0: memory@d0000000 {
+		reg = <0xd0000000 DT_SIZE_M(1)>;
+	};
+};
+
+&lpuart2 {
+	status = "okay";
+	current-speed = <115200>;
+	/* clocks = <&ccm IMX_CCM_UART4_CLK 0x6c 24>; */
+	pinctrl-0 = <&uart2_default>;
+	pinctrl-names = "default";
+};
```

Comment:
GPIOs are not enabled but listed as supported feature. Either remove from docs or enable them here.

Reply:
Removed GPIOs from doc and the *.yaml files. If there is more time for testing, I will add those features step-by-step in follow-up PRs.